README.md

```diff
@@ -53,14 +53,68 @@ On OSX/Windows you can't run Docker natively. To allow the Makefile to write the
 Run `make test` like this: `LOCAL_USER_ID=1000 LOCAL_GROUP_ID=100 make test-containerized`
 
 
+
+## IPv6 Usage
```

Comment:
As a user, can I use _just_ ipv6? Or do I always use ipv4 and ipv6? Can you answer this question by updating the readme?